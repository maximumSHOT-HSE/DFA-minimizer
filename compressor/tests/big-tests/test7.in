15 75 5
1 4 0
1 7 1
1 6 2
1 4 3
1 8 4
2 5 0
2 7 1
2 12 2
2 1 3
2 3 4
3 12 0
3 12 1
3 9 2
3 7 3
3 3 4
4 13 0
4 6 1
4 5 2
4 6 3
4 9 4
5 3 0
5 5 1
5 12 2
5 12 3
5 4 4
6 5 0
6 3 1
6 14 2
6 3 3
6 13 4
7 11 0
7 3 1
7 5 2
7 3 3
7 2 4
8 5 0
8 2 1
8 3 2
8 12 3
8 12 4
9 8 0
9 13 1
9 1 2
9 1 3
9 9 4
10 12 0
10 12 1
10 11 2
10 15 3
10 7 4
11 10 0
11 12 1
11 4 2
11 13 3
11 15 4
12 1 0
12 2 1
12 15 2
12 15 3
12 13 4
13 15 0
13 2 1
13 1 2
13 10 3
13 8 4
14 2 0
14 3 1
14 1 2
14 2 3
14 2 4
15 12 0
15 1 1
15 6 2
15 14 3
15 4 4
9
11
11 12 14 11 9 6 12 7 14 6 6 
