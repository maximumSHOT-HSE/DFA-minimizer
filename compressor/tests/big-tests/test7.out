15 75 5
12 1 0
15 1 1
9 1 2
13 1 2
14 1 2
2 1 3
9 1 3
14 2 0
8 2 1
12 2 1
13 2 1
14 2 3
7 2 4
14 2 4
5 3 0
6 3 1
7 3 1
14 3 1
8 3 2
6 3 3
7 3 3
2 3 4
3 3 4
1 4 0
11 4 2
1 4 3
5 4 4
15 4 4
2 5 0
6 5 0
8 5 0
5 5 1
4 5 2
7 5 2
4 6 1
1 6 2
15 6 2
4 6 3
1 7 1
2 7 1
3 7 3
10 7 4
9 8 0
1 8 4
13 8 4
3 9 2
4 9 4
9 9 4
11 10 0
13 10 3
7 11 0
10 11 2
3 12 0
10 12 0
15 12 0
3 12 1
10 12 1
11 12 1
2 12 2
5 12 2
5 12 3
8 12 3
8 12 4
4 13 0
9 13 1
11 13 3
6 13 4
12 13 4
6 14 2
15 14 3
13 15 0
12 15 2
10 15 3
12 15 3
11 15 4
9
6
6 7 9 11 12 14 
